userapi/src/main/java/fk/prof/userapi/http/UserapiApiPathConstants.java

```diff
@@ -1,13 +1,31 @@
 package fk.prof.userapi.http;
 
 public final class UserapiApiPathConstants {
+    private UserapiApiPathConstants() {
+    }
 
-  private UserapiApiPathConstants() { }
+    public static final String POLICY_API_PREFIX = "/policy";
+    public static final String LIST_POLICY_API_PREFIX = "/list/policy";
```

Comment:
`public static final String POLICIES_PREFIX = "/policies";`
Similar to how we do for profiles. Avoid verbs (like list) in rest api names. Use http verbs(get/post) and nouns in api name to indicate the intended effect

Reply:
Made the change [here](https://github.com/Flipkart/fk-prof/commit/56e473b644ac6105074c49fe6fef0ddb18bf4405l)